scripts/joinmarket-qt.py

```diff
@@ -1473,18 +1486,74 @@ def closeEvent(self, event):
         else:
             event.ignore()
 
+    def makerManager(self):
+        if not self.wallet_service:
+            JMQtMessageBox(self, "No wallet loaded.",
+                           mbtype='crit', title="Error")
+            return
+        action_fn = self.stopMaker if self.maker_running else self.startMaker
+        self.makerDialog = MakerDialog(action_fn, self.maker_running,
+                                       self.maker_settings)
+
+    def toggle_non_maker_function(self):
+        """ While maker is running we prevent actions
+        to do coinjoins as taker or to load or alter the wallet
+        or change settings.
+        TODO These restrictions can be relaxed after analysis.
+        """
+        for action in [self.loadAction, self.generateAction, self.recoverAction]:
+            action.setEnabled(not self.maker_running)
+        for tab in [self.centralWidget().widget(x) for x in [1,2]]:
+            tab.setEnabled(not self.maker_running)
+
+    def startMaker(self):
+        mle = self.makerDialog.maker_settings_ql
+        offertype = "reloffer" if mle[0][1].currentText() == "Relative fee" else "absoffer"
+        cjabsfee = int(mle[1][1].text())
+        cjrelfee = float(mle[2][1].text())
+        txfee = int(mle[3][1].text())
+        minsize = int(mle[4][1].text())
+        # keep track of currently chosen settings for next display of dialog:
+        self.maker_settings = [mle[0][1].currentText(), cjabsfee, cjrelfee, txfee, minsize]
+        # note: a quirk of JM maker code, relative fees are assumed to come in as string:
+        self.makerfactory = ygstart(self.wallet_service, txfee, cjabsfee, str(cjrelfee),
+                                        offertype, minsize, rs=False)
+        self.maker_running = True
+        self.setMakerBtn()
+        self.toggle_non_maker_function()
+        self.makerDialog.close()
+
+
+    def setMakerBtn(self):
+        if self.maker_running:
+            self.makerbtn.setIcon(read_QIcon("greencircle.png"))
+            self.makerbtn.setToolTip("Maker running: click to manage")
+        else:
+            self.makerbtn.setIcon(read_QIcon("reddiamond.png"))
+            self.makerbtn.setToolTip("Maker not running: click to manage.")
```

Comment:
This is just pedantic, but one tooltip ends with '.' the other does not. Since they are "symmetrical" it would be nice to have them consistent.